### DISCUSSION TITLE
Is this correct way to upgrade to the latest version?

### DISCUSSION THREAD

**Discussion by pacez** [verdaccio/verdaccio#3215] [Q&A] [answered]
```shell
npm i -g verdaccio@latest
```
Whether reconfiguration is required after the upgrade？

**Comment by juanpicado** [comment, marked as answer]
Yes if you refer to 5.x ... 
Upgrade from .. ? Could you elaborate?

**Reply by pacez** [reply]
Thanks for your reply,  I want to upgrade from 5.8.0 to  latest(5.11.0).

**Reply by juanpicado** [reply]
Nothing special, since v5.0.0 changes are minimum. You can upgrade even future 5.x.x with no pain because I never break unless is a major release which will have a migration guideline eventually.

**Reply by pacez** [reply]
Thanks again, I go to the upgrade.  ^_^
